examples/bluetooth/bluedroid/classic_bt/bt_spp_initiator/main/main.c

```diff
@@ -189,7 +189,7 @@ static void esp_spp_cb(esp_spp_cb_event_t event, esp_spp_cb_param_t *param)
              * rather than in this callback directly. Since the printing takes too much time, it may stuck the Bluetooth
              * stack and also have a effect on the throughput!
              */
-            ESP_LOGI(SPP_TAG, "ESP_SPP_WRITE_EVT len:%d handle:%d cong:%d", param->write.len, param->write.handle,
+            ESP_LOGI(SPP_TAG, "ESP_SPP_WRITE_EVT len:%d handle:%lu cong:%d", param->write.len, param->write.handle,
```

Comment:
1. For this type, I think it is better to use `PRIu32` instead of direct `%lu`.
`ESP_LOGI(SPP_TAG, "ESP_SPP_WRITE_EVT len:%d handle:%"PRIu32" cong:%d", param->write.len, param->write.handle,`
2. Based on this, there is the same problem in another code, and a similar modification. 
(examples/bluetooth/bluedroid/classic_bt/bt_spp_acceptor/main/main.c)
line 110: `ESP_LOGI(SPP_TAG, "ESP_SPP_DATA_IND_EVT len:%d handle:%"PRIu32,`